core/src/main/scala/org/bitcoins/core/protocol/ln/LnInvoiceTags.scala

```diff
@@ -66,23 +96,62 @@ object LnInvoiceTag {
   case class ExpiryTimeTag(u64: UInt64) extends LnInvoiceTag {
     override val prefix: LnTagPrefix = LnTagPrefix.ExpiryTime
 
-    override val bytes: ByteVector = u64.bytes
+    override val bytes: ByteVector = {
+      encodeNumber(u64.toInt)
+    }
+
+    override def dataToBech32: String = {
+      //bytes is already in base5, so need to to decode again
+      Bech32.encode5bitToString(bytes)
+    }
   }
 
   case class CltvExpiryTag(u64: UInt64) extends LnInvoiceTag {
     override val prefix: LnTagPrefix = LnTagPrefix.CltvExpiry
 
-    override val bytes: ByteVector = u64.bytes
+    override val bytes: ByteVector = {
+      encodeNumber(u64.toInt)
```

Comment:
This isnt safe unless we add some `require` statements to the the` CtlvExpiryTag` and `ExpiryTimeTag`.